worker-markup-core/src/main/java/com/github/cafdataprocessing/worker/markup/core/MarkupHeadersAndBody.java

```diff
@@ -166,7 +167,7 @@ else if (line.contains(": ")) {
                 else if (line.contains(":*")) {
                     bodyIndex++;
                     addStandardisedHeader(nattyParser, headersElement, lines, line, ":\\*");
-                }else {
+                } else {
```

Comment:
Remove whitespace changes from the commit. These are good to get in but they should be in their own commit, that way if we back out a feature we don't also lose good formatting changes.